AWSAppSyncClient/Internal/AWSPerformMutationOperation.swift

```diff
@@ -6,12 +6,19 @@
 
 import Foundation
 
+/// This class(operation) is responsible for executing mutations when they are submitted to the operation queue
+/// from application side and not when loading from cache. If the app is killed and restarted, these operations
+/// are lost. To ensure they still are sent, they are loaded from the cache and a different type of operation is
+/// created: `AWSPerformOfflineMutationOperation`.
 final class AWSPerformMutationOperation<Mutation: GraphQLMutation>: AsynchronousOperation, Cancellable {
```

Comment:
While we're in here, what about renaming these mutation classes to something more descriptive like `CachedMutationOperation` and `SessionMutationOperation` or similar? Basically, something to clearly differentiate between mutations that were invoked during the current app session and mutations loaded from cache, which therefore lose callback context?

Reply:
I like the idea. I was slightly confused and thus the documentation. But renaming should alleviate some issues.